### ISSUE TITLE
Addon not updating MAL watchlists

### ISSUE THREAD

**Issue by ferrazpedro** [SageTendo/mal-stremio-addon#20]
Good afternoon!

I'm having a problem that whenever I try to watch any anime, Streamio gives an error prompt about an subtitle extension and the watched episode never gets marker in MAL. Is there something I can do about it?

Thanks in advance!

**Comment by SageTendo** [comment]
Hi there. Can you tell me what you're trying to watch or if it's not working for anything you watch? Are the shows you want to watch, in one of your anime lists (watching, plan to watch, on hold)? 

The add-on will only track shows in those lists, so if you want to mark a new show, you'd have to add it manually to your list first. This is to avoid adding a lot of shows that you only watch one episode of, and decide to not continue watching.

**Comment by ferrazpedro** [comment]
This is happening with any anime, even the ones in my watch lists! And besides that, I'm having this error any time I open one:

![image](https://github.com/user-attachments/assets/dc18e952-dd3c-43e2-8eee-ab7d5fe61eb3)

**Comment by SageTendo** [comment]
Okay after some digging around, it seems there's an issue with the anime database API I'm calling. I'll see if there's anything I can do about it, maybe talk to the dev or look into forking their repo and runnning my own.
This might take a while, as I have a lot of work at the moment. Sorry for the inconvenience.

**Comment by SageTendo** [comment]
> This is happening with any anime, even the ones in my watch lists! And besides that, I'm having this error any time I open one:
> 
> ![image](https://private-user-images.githubusercontent.com/37643357/361038773-dc18e952-dd3c-43e2-8eee-ab7d5fe61eb3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ1MjI5MTAsIm5iZiI6MTcyNDUyMjYxMCwicGF0aCI6Ii8zNzY0MzM1Ny8zNjEwMzg3NzMtZGMxOGU5NTItZGQzYy00M2UyLThlZWUtYWI3ZDVmZTYxZWIzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI0VDE4MDMzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMyNmQ4ZWI4ZGFiNTQ0MjJjYzYwY2Q1ZTYxYWEyMGIwMDNhOTI1NjRkZDU5NTc5MjQ2YjQyZWVlMjNiNGQ1N2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.74PAqKw_pkJ__ly-l_seUFJcDKE14oG-4CAzDJDlI8U)

That error happens because the app thinks its downloading a subtitle file but gets nothing. The subtitles API is used to check what is being watched and to sync it with MAL.

**Comment by Jeffison2005** [comment]
hello, wanted to say that i'm also having this problem, i've watched the episode fully through and it didn't update it. tried it on an anime i have on-hold and again fully watched the episode till the next one started and that didn't update it to the watching tab. so yeah.

information gets sent to stremio but stremio isn't sending information to MAL (or mal just isn't receiving it)

**Comment by SageTendo** [comment]
> hello, wanted to say that i'm also having this problem, i've watched the episode fully through and it didn't update it. tried it on an anime i have on-hold and again fully watched the episode till the next one started and that didn't update it to the watching tab. so yeah.
> 
> information gets sent to stremio but stremio isn't sending information to MAL (or mal just isn't receiving it)

https://github.com/SageTendo/mal-stremio-addon/issues/20#issuecomment-2308441688
Hi, I've found the problem and it's something I can't fix myself. I'm waiting on the developer of that service. You can take a look at the previous comment for more information.

In the meantime I'm working on running my own anime database to avoid the issue again.

**Comment by Jeffison2005** [comment]
that's fine, going to MAL and manually doing it won't be the death of me, but excited when it starts working!!!

**Comment by SageTendo** [comment]
> that's fine, going to MAL and manually doing it won't be the death of me, but excited when it starts working!!!

The issue seems to have been resolved on the developer's end. Everything should be working as usual. I'll still leave this issue open for the local database I plan to work on.
If there's anything else, feel free to let me know.
